modules/templates-rails-writing-application.adoc

```diff
@@ -13,7 +13,7 @@ Rails gem first. Then you can proceed with writing your application.
 +
 ----
 $ gem install rails
-Successfully installed rails-4.3.0
+Successfully installed rails-4.4.0
```

Comment:
It's almost certainly a coincidence that `rails` has the same version as OCP 4.3 did.